addons/mail/static/src/core/web/chatter.js

```diff
@@ -99,6 +106,10 @@ export class Chatter extends Component {
             showActivities: true,
             /** @type {import("@mail/core/common/thread_model").Thread} */
             thread: undefined,
+            isSearchOpen: false,
+            searchMessages: [],
+            /** @type {string|undefined} */
+            searchTerm: undefined,
```

Comment:
As said in a previous comment, `searchMessages`/`searchTerm` will be moved in `useMessageSearch()` reactive component hook